developer/docs/arvo/network.md

```diff
@@ -15,38 +15,43 @@ All we've written up until now are just shell commands that produce a value and
 then disappear. To listen for and receive messages from other planets, we'll
 need an app. Let's look at a very simple one, `echo.hoon`:
 
-    ::  Accepts any noun from dojo and prints it out
-    ::
-    ::::  /===/app/echo/hoon
-      ::
-    !:                          :: enables debug printfs
-    |%                          :: forms a core with subject as payload
-    ++  move  {bone card}       :: defines echo app's arvo move mold
-    ++  card  $%  $~            :: defines card (data) mold as null tagged union 
-              ==                :: end of mold definition; see $%
-    --                          :: end of card arm definition
-    ::                          ::
-    |_  {bow/bowl $~}           :: forms a core with bowl (app state) as sample
-    ++  poke-noun               :: begin definition of poke-noun arm
-      |=  non/*                 :: forms a gate which accepts any noun
-      ^-  {(list move) _+>.$}   :: casts product to mold of moves and context
-      ~&  echo+noun+non         :: debug printf, i.e. '[%echo %noun *]'
-      [~ +>.$]                  :: produce a cell of empty list and our state
-    --                          :: end of poke-noun arm definition
+```
+::  Accepts any noun from dojo and prints it out        ::  1 
+::                                                      ::  2 
+::::  /===/app/echo/hoon                                ::  3 
+  ::                                                    ::  4 
+!:                                                      ::  5 
+|%                                                      ::  6 
++=  move  {bone card}                                   ::  7 
```

Comment:
Yeah, I'll make the examples styling more consistent since now I think I actually know all the things to check/change. Then we can make all this code consistent with that.